### ISSUE TITLE
hide print out of the github password in the error log

### ISSUE THREAD

**Issue by nserebry** [ashwanthkumar/gocd-build-github-pull-requests#115]
Hi,
Setup GitHub.pr plugin on the go server, with Go Version: 17.10.0
create a pipeline with Github material 
make the material update fail, 
When the material update check fails below log message shown in go-server
`Interaction with plugin with id 'github.pr' implementing 'scm' extension failed while requesting for 'checkout'. Reason: [The plugin sent a response that could not be understood by Go. Plugin returned with code '500' and the following response: '"Exception (Process exited with an error: 128 (Exit value: 128)) Occurred: [git, clone, --branch\u003dmaster, https://user-name:password@github.com/XXXX/web, /var/lib/docker/godata-agent-1/pipelines/web-pr-tests/web-repo-pr] - null"']`

Currently Gocd prints out the github password in the error log above, should be a way to encrypt it or just not display.

Thank you,
N

**Comment by ashwanthkumar** [comment]
This was supposed to be fixed as part of https://github.com/ashwanthkumar/gocd-build-github-pull-requests/pull/107. Can you please upgrade the plugin version you're using and try again?

**Comment by nserebry** [comment]
@ashwanthkumar we are using Github Pull Requests Builder 1.3.4 [github.pr], so it looks to me that fix should be in it ?

**Comment by ashwanthkumar** [comment]
Yes it should be as part of that release. Oh wait, can you please let me know what's the Github url you're using while configuring the Material?

**Comment by nserebry** [comment]
this is a private repo:   [url=https://github.com/ORG/project_name] 
<img width="854" alt="edit_pipeline_-_go" src="https://user-images.githubusercontent.com/5048814/34055966-3a5a7234-e186-11e7-90ef-84ef95b7e20b.png">

**Comment by nserebry** [comment]
Hi @ashwanthkumar  just following up
